core/tests/protractor_utils/general.js

```diff
@@ -24,6 +24,9 @@ var dragAndDropScript = require('html-dnd').code;
 var action = require('../protractor_utils/action.js');
 
 var dragAndDrop = async function(fromElement, toElement) {
+  waitFor.presenceOf(
+    dragAndDropScript,
+    'Drag and drop script taking too long to load');
```

Comment:
Sorry I wasn't clear. I meant we should wait for the fromElement and toElement to be visible